Dockerfile

```diff
@@ -1,10 +1,11 @@
+FROM golang:1.8
+RUN go get -u github.com/vimukthi-git/beanstalkg
+WORKDIR /src/github.com/vimukthi-git/beanstalkg
+COPY . /src/github.com/vimukthi-git/beanstalkg
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o /beanstalkg .
+
 FROM scratch
```

Comment:
Shall we start from `https://hub.docker.com/r/centurylink/ca-certs/` image and remove the `ca-certificates.crt` file from the repo and the DockerFile?